src/AspNetCore/Src/forwardinghandler.cxx

```diff
@@ -1412,6 +1415,7 @@ FORWARDING_HANDLER::OnExecuteRequestHandler(
             {
                 if (!fCompletionExpected)
                 {
+                    m_pW3Context->SetRequestHandled();
```

Comment:
And here?

Reply:
this is standard IIS pattern since win7. We should follow it